relational_operators/TextScanOperator.cpp

```diff
@@ -18,6 +18,11 @@
  **/
 
 #include "relational_operators/TextScanOperator.hpp"
+#include "relational_operators/RelationalConfig.h"
```

Comment:
@tarunbansal As I suggested before, please add comments `// For QUICKSTEP_HAVE_UNISTD.` after the head file.